css/_base.scss

```diff
@@ -8,20 +8,21 @@
     .page__title {
       text-align: center;
       font-size: 40px;
-      padding-bottom: 45px;
+    }
       
-      .page__nav {
-        padding-top: 20px;
-        font-size: 20px;
-
-        .page__nav--link {
-          transition: color 250ms;
-          cursor: pointer;
-          text-decoration: underline;
+    .page__nav {
+      padding-top: 20px;
```

Comment:
Since you want space between elements, I would use margin instead of padding here. 

```scss
.page__nav {
  margin: 40px 0;
}
```